MapboxGeocoder/MBPlacemark.swift

```diff
@@ -152,7 +152,7 @@ open class Placemark: NSObject, Codable {
     /**
      A subset of the `properties` object on a GeoJSON feature suited for Geocoding results.
      */
-    fileprivate var properties: Properties?
+    public var properties: Properties?
```

Comment:
> along with exposing `Placemark.properties`, which I don't think should be exposed after a second thought because we have aliases to most of these properties. 🤔 

Yeah, that was my original intention with this architecture. The Geocoding API has always been quite strictly oriented around GeoJSON, but I’ve always felt that clients of the library need a stronger, less general-purpose type system. Which properties don’t have dedicated properties?